### DISCUSSION TITLE
"WARNING rustls failed to parse DER certificate" when running turbo command

### DISCUSSION THREAD

**Discussion by LL782** [vercel/turborepo#7404] [Help] [answered]
Every time I try to use the turbo CLI I get the following warning from rustls

```
WARNING  rustls failed to parse DER certificate MissingOrMalformedExtensions
```

and then there's a details of the Certificate like...

`Certificate(b"0\x82\x03l0\x82\x02T\xa0\x03\x02\x01\x02\x02\x06\x01y!\xbf\x0f\xda0\r\x06[...]")`

_Note: `[...]` is me not included the whole thing_

**Comment by mehulkar** [comment]
I believe this was fixed in https://github.com/vercel/turbo/pull/7357, please try the latest release!

**Reply by LL782** [reply]
Yes thank you!

**Comment by anthonyshew** [comment, marked as answer]
Hoping that the lack of communication here means its fixed! If not, please open a new issue.
